Add LCOV plugin to default channel.

thraex-aquator opened the pull request:
<!--
The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
Patience padawan!

You can request a review from @packagecontrol-bot.
Please ensure the reviews pass and follow any instructions.

Please provide some information via this checklist,
feel free to remove what't not applicable.
-->

- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **

My package is showing covered/missed lines based on an lcov.info file.

There are no packages like it in Package Control.


<!-- 
*)   Unless it definitely really needs them,
     they apply to the cursor's context
     and their visibility is conditional.
     Space in this menu is limited!
**)  There aren't enough keys for all packages,
     you'd risk overriding those of other packages.
     You can put commented out suggestions in a keymap file, 
     and/or explain how to create bindings in your README.
***) We have hundreds of color schemes,
     and plenty of scopes to make any syntax work. 

For bonus points also considered how the review guidelines apply to your package:
https://github.com/wbond/package_control_channel/wiki#reviewing-a-package-addition

For updates to existing packages:
If your package isn't using tag based releases,
please switch to tags now.
 -->

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

kaste commented:
LGTM from here.  I think it requires Sublime Text `>=4000` though as you target (rightfully) python 3.8.  Note that you don't need to instantiate `Listener()`; Sublime does it for you automatically.

thraex-aquator commented:
Thanks for the heads-up! Bumped the version to 4000, also removed the extra initialization and reduced the scope of the observer while I was already there. Moved the 1.0.0 tag to the current head as it was never really released before.

braver commented on repository/l.json:
```diff
@@ -673,6 +673,18 @@
+		{
+			"name": "LCOV",
+			"details": "https://gitlab.com/aquator/sublime-lcov",
+			"readme": "https://gitlab.com/aquator/sublime-lcov/-/raw/master/README.md",
```
```suggestion
```

Since that's just the default readme location, you can omit it.

thraex-aquator replied:
Thanks, did not know that, I copied from another gitlab based plugin.
Removed the readme line.

braver commented:
Ugh, our tools don't want to run on GitLab repos. It looks good enough to just run with it and see what happens though 👍🏻

thraex-aquator commented:
Do I need to do anything else regarding being on gitlab, or you will handle it? (The suggested change is there)

braver commented:
Ok, let’s see if it works. Maybe we need to add some metadata here, but we just need to check in a couple of hours.

kaste commented:
✋ packagecontrol.io doesn't pick it up

braver commented:
@kaste thanks reporting that. It seems to be a more widespread issue than just this package. I have no idea what might cause it. Of 5516 packages, 6 are on GitLab, half of which are missing. 🤷🏻 

Will look into it: https://github.com/wbond/package_control_channel/issues/9020
